lib/rubygems/sigstore/gem_verifier.rb

```diff
@@ -12,7 +12,7 @@ def initialize(gemfile:, config:)
 
   def run
     rekor_api = Gem::Sigstore::RekorApi.new(host: config.rekor_host)
-    entries = rekor_api.where(data_digest: gemfile.digest)
+    entries = rekor_api.where(data_digest: gemfile.digest) # TODO: we should only pass on the entries where body.kind == "rekord"
```

Comment:
\nit the comment could live in an issue, which would make it easier for others to pick it up 🙂

Reply:
The latter.  The refactor itself or a follow up PR will ensure that only rekords are retained.